src/feed/Feed.ts

```diff
@@ -335,7 +337,6 @@ class Feed {
                 if (response.error) {
                     const { message: text } = response.error as TError;
                     this.loader.hide();
-                    this.paginationLoader.updateOnPagination(false);
```

Comment:
A small note just in case: This line is not needed anymore here because in case response.error === true, it will quit _getPaginationData() upon `return` at line 346 and execute `this.paginationLoader.updateOnPagination(false)` at line 304 instead.